### PR TITLE
Unbook a laundry time

### PR DESCRIPTION
**Pull request by Sajaas**
## PT link: https://www.pivotaltracker.com/story/show/151365444 ##
* User can unbook their laundry time

### CODE REVIEW

File: app/controllers/laundry_rooms_controller.rb [hunk 1]
```diff
@@ -25,4 +25,11 @@ def create_booking
       redirect_to root_path
     end
   end
+
+  def delete_booking
+    active_bookings = current_user.bookings.detect { |booking| booking.time == params[:time] }
```
Comment (tochman) [inline app/controllers/laundry_rooms_controller.rb]: We are dealing with one perticular booking here, right? Why plural form on the variable name?

File: app/views/laundry_rooms/index.html.erb [hunk 2]
```diff
@@ -8,6 +8,7 @@
               <% if booking[:time] > (DateTime.now - 5.hours) %>
                 <ul>
                   <li><%= booking[:time].beginning_of_hour.to_formatted_s(:short) %></li>
+                  <%= link_to 'Unbook', laundry_room_delete_booking_path(laundry_room_id: @laundry_rooms.first, time: booking[:time]) %>
```
Comment (tochman) [inline app/views/laundry_rooms/index.html.erb]: Don't we have the LaundryRoom instance saved in a variable at this point? What is the danger of hardcoding the instance to `@laundry_rooms.first`?

**Review by tochman — CHANGES_REQUESTED** [review]

File: app/controllers/laundry_rooms_controller.rb (hunk 1 above)
Comment (Sajaas) [reply app/controllers/laundry_rooms_controller.rb]: Ok.

File: app/views/laundry_rooms/index.html.erb (hunk 2 above)
Comment (Sajaas) [reply app/views/laundry_rooms/index.html.erb]: Where do we have the LaundryRoom instance saved? 

What I can tell we only have all the laundry rooms set, in the LaundryRoom controller index action. Supporting multiple laundry rooms might be a different User story in the future, wouldn't you say? We did agree to this early in the design sprint. 

Please correct me if I'm misunderstanding your comment.

Comment (Sajaas) [reply app/views/laundry_rooms/index.html.erb]: Using booking.bookable_id

**Review by tochman — APPROVED** [review]

**Pull request merged**